kubernetes/deployment.template.yaml

```diff
@@ -41,7 +40,7 @@ data:
   CCD_UI_BASE_URL: "https://ccd-case-management-web-${SERVICE_FQDN}"
   FPL_LOCAL_AUTHORITY_EMAIL_TO_CODE_MAPPING: "swansea.gov.uk=>SA;hillingdon.gov.uk=>HN"
   FPL_LOCAL_AUTHORITY_CODE_TO_NAME_MAPPING: "SA=>Swansea City Council;HN=>London Borough Hillingdon;PCC=>Portsmouth City Council;STF=>Staffordshire County Council;SCC=>Southampton City Council"
-  FPL_LOCAL_AUTHORITY_USER_MAPPING: "SA=>29,30,31;HN=>32,33"
+  FPL_LOCAL_AUTHORITY_USER_MAPPING: "SA=>31,32,33;HN=>34,35"
```

Comment:
I've got HIllingdon as 34 and 36 in my db... Cafcass is 35. Are you sure this is correct?

![Screen Shot 2019-05-08 at 09 17 45](https://user-images.githubusercontent.com/29975020/57360538-4cc99300-7172-11e9-8347-01b15291b92b.png)